Installing-and-Configuring/Building.md

```diff
@@ -54,8 +54,9 @@ git clone https://github.com/osresearch/heads
 cd heads
 ```
 
-Builds of Heads [should be](https://github.com/osresearch/heads/issues/734)
- reproducible, which means that Heads will build in the exact same way on
+Builds of Heads should be unless reproducible issues are currently known,
```

Comment:
Builds of Heads should be unless reproducible issues are currently known -> Builds of Heads should be reproducible unless issues are currently known

Reply:
![36rfik](https://user-images.githubusercontent.com/553624/95771376-757c8300-0caa-11eb-98ef-c1d0ad876d47.jpg)